core/src/main/java/org/radargun/DistStageAck.java

```diff
@@ -54,9 +58,16 @@ public void setRemoteException(Throwable remoteException) {
             for (StackTraceElement ste : stackTraceElements) {
                sb.append("\n\t at ").append(ste.toString());
             }
+            sb.append("\nsuppressed: ");
+            sb.append(remoteException.getSuppressed().toString());
```

Comment:
I think you'll need to use either Arrays.toString or iterate over the elements and print individual StackTraceElements if you want more detailed message (I'd opt for option no. 2).